packages/cli/src/hooks/command_not_found/command_moved.ts

```diff
@@ -9,6 +9,8 @@ const hook: Hook<"command_not_found"> = async function (opts) {
     process.stdout.write(
       `You can use it like: ${chalk.greenBright(`swanky contract ${opts.id} contract_name`)}\n`
```

Comment:
I think this file should be renamed as it handles more scenarios than before.

Reply:
I think so too.

`packages/cli/src/hooks/command_not_found.ts` and modify package.json like below?

```
"hooks": {
      "command_not_found": [
        "./dist/hooks/command_not_found"
      ]
    }
```